modules/logging-release-notes-5-9-0.adoc

```diff
@@ -0,0 +1,79 @@
+
+[id="logging-release-notes-5-9-0-bug-fixes"]
+== Bug Fixes
+* Before this update, the {logging} must-gather could not collect any logs on a FIPS-enabled cluster. With this update, a new `oc` client is available in `cluster-logging-rhel9-operator`, and must-gather works properly on FIPS clusters. (link:https://issues.redhat.com/browse/LOG-4403[LOG-4403])
+
+* Before this update, the LokiStack ruler pods could not format the IPv6 pod IP in HTTP URLs used for cross-pod communication. This issue caused querying rules and alerts through the Prometheus-compatible API to fail. With this update, the LokiStack ruler pods encapsulate the IPv6 pod IP in square brackets, resolving the problem. Now, querying rules and alerts through the Prometheus-compatible API works just like in IPv4 environments. (link:https://issues.redhat.com/browse/LOG-4709[LOG-4709])
+
+* Before this fix, the YAML content from the {logging} must-gather was exported in a single line, making it unreadable. With this update, the YAML white spaces are preserved, ensuring that the file is properly formatted. (link:https://issues.redhat.com/browse/LOG-4792[LOG-4792])
+
+* Before this update, when the `ClusterLogForwarder` was enabled, the {clo} could run into a nil pointer exception when `ClusterLogging.Spec.Collection` was nil. With this update, the issue is now resolved in the {clo}. (link:https://issues.redhat.com/browse/LOG-5006[LOG-5006])
+
+* Before this update, in specific corner cases, replacing the `ClusterLogForwarder` status field caused the `resourceVersion` to constantly update due to changing timestamps in `Status` conditions. This condition led to an infinite reconciliation loop. With this update, all status conditions synchronize, so that timestamps remain unchanged if conditions stay the same. (link:https://issues.redhat.com/browse/LOG-5007[LOG-5007])
+
+* Before this update, there was an internal buffering behavior to `drop_newest` to address high memory consumption by the collector resulting in significant log loss. With this update, the behavior reverts to using the collector defaults. (link:https://issues.redhat.com/browse/LOG-5123[LOG-5123])
+
+* Before this update, the {loki-op} `ServiceMonitor` in the `openshift-operators-redhat` namespace used static token and CA files for authentication, causing errors in the Prometheus Operator in the User Workload Monitoring spec on the `ServiceMonitor` configuration. With this update, the {loki-op} `ServiceMonitor` in `openshift-operators-redhat` namespace now references a service account token secret by a `LocalReference` object. This approach allows the User Workload Monitoring spec in the Prometheus Operator to handle the {loki-op} `ServiceMonitor` successfully, enabling Prometheus to scrape the {loki-op} metrics. (link:https://issues.redhat.com/browse/LOG-5212[LOG-5165])
```

Comment:
Should this be `ServiceMonitor` CR or object?

Reply:
I did not see an instance where engineering has used either term for `ServiceMonitor`. Reference PR: https://github.com/grafana/loki/pull/12216 or https://issues.redhat.com/browse/LOG-5212. I'm guessing we can leave it how it's they've described it.